editions/tw5.com/tiddlers/widgets/TranscludeWidget.tid

```diff
@@ -42,7 +42,7 @@ Modern mode is recommended for use in new applications.
 
 | !Attribute |<| !Description |
 | !(modern) | !(legacy) |~|
-|$variable |- |Name of the variable to transclude |
+|$variable |- |Name of the [[variable|Variables]] or [[procedure|Procedures]] to transclude |
```

Comment:
Some lines below it says: `[. . .] (note that procedures, custom widgets and macros are all special types of variable)` So I think it should be extended to the full list @Jermolene .. Is there something else, which is transcluded as variables?